README_template.md

```diff
@@ -0,0 +1,40 @@
+# Project Name
+One paragraph description of the project goes here.
+
+## Written in
+Programming Language
+Using the folowing libraries:
+* [pandas](http://pandas.pydata.org/)
+* [numpy](http://www.numpy.org/)
+* ...
+
+## Installation
+A step by step guide on how to get your project working on a new machine.
+Note: Write this with someone else, using their machine. It is easy to forget
+dependencies.
+
+End this section with a simple example of how to get your project to do
+something useful.
+
+## Tests
+Explain how to run the tests for this program.
+### Automated Tests?
+If you have a set of automated tests how does a new user run them?
+### End-to-end tests
+What do these test? Why?
+
+## Authors
+A list of people who wrote the project.
+(Maybe a link to a list of contributors)
+
```

Comment:
Can we add a section on data - what data does the project use and where was it sourced from?